### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Benjamin Kocher

### DISCUSSION THREAD

**Discussion by BKo1706** [heig-vd-dai-course/heig-vd-dai-course#342] [Show and tell]
https://github.com/BKo1706/intro_DAI

**Comment by ludelafo** [comment]
Looks good to me, thanks!

One tip: you can use code blocks to format code as mentioned in the course material: https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#code-blocks or `inline code` with the command you want to highlight between `` ` `` (backtick) characters.

It makes the reading of your commands easier. :)
